ParisBNF/abb/BNFabb156.xml

```diff
@@ -419,7 +419,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                        <item xml:id="a5">
                           <locus from="67rb" to="67v"/>
                           <desc type="GuestText" corresp="LIT2332serata" cert="low">A short Śǝrʿāta beta krǝstiyān text in Gǝʿǝz, 
-                             probably incomplete, possibly somehow related to <ref target="#a1 #a2 #3">Additional notes 1, 2 or 3</ref>.</desc>
```

Comment:
why not links? they were wrong, it should be
`Additional notes <ref target="#a1">1</ref>, <ref target="#a2">2</ref> or <ref target="#a3">3</ref>`

Reply:
ok